src/domain/safe/entities/schemas/safe.schema.ts

```diff
@@ -0,0 +1,26 @@
+import { JSONSchemaType } from 'ajv';
+import { Safe } from '../safe.entity';
+
+export const safeSchema: JSONSchemaType<Safe> = {
+  type: 'object',
+  properties: {
+    address: { type: 'string' },
+    nonce: { type: 'number' },
+    threshold: { type: 'number' },
+    owners: { type: 'array', items: { type: 'string' } },
+    masterCopy: { type: 'string' },
+    modules: { type: 'array', nullable: true, items: { type: 'string' } },
+    fallbackHandler: { type: 'string' },
+    guard: { type: 'string' },
+    version: { type: 'string', nullable: true },
```

Comment:
I think that version is not `nullable`.

Reply:
Yes you are right. The current CGW has it as nullable https://github.com/safe-global/safe-client-gateway/blob/56fa8a32b2284c64af2c5caccb17bfa04aa207fc/src/providers/info.rs#L64

I will change it here.

Reply:
See [22b5798](https://github.com/5afe/safe-client-gateway-nest/pull/116/commits/22b5798bcd7c6e17e4788c078b3dd1ae0900b639)